text/3437-implementable-trait-alias.md

```diff
@@ -287,6 +287,25 @@ impl Frobber for MyType {
 
 Trait aliases are `unsafe` to implement iff the underlying trait is marked `unsafe`.
 
+Implementable trait aliases can also be used with trait-qualified and fully-qualified method call syntax. When used this way,
```

Comment:
Why only function calls? This should probably state that they can be used in general as the qualifying trait for a qualified path, so that you address consts, functions and type aliases altogether.

Reply:
Yes, that would make sense, I will fix it.